lighthouse-core/test/chromium-web-tests/batch-test-pages-devtools.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+##
```

Comment:
Some notes up top on how to use would be useful (using the checked in `urls.txt` as an example)